README.md

```diff
@@ -406,6 +406,31 @@ If '--logfile' used when running bot "logfile": "pycryptobot.log" line in config
 "fileloglevel" and "consoleloglevel" can get one of 'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'
 For further detail in log levels: https://docs.python.org/3/library/logging.html#logging-levels
 
+# Binance BNB fee calculation
+
+Binance trading fees can be paid as Binance Coin (BNB).
+To get bot calculate buy fee pais as BNB need to set "feeasset" key in config.json.
```

Comment:
To get the bot to calculate the buy fees paid as BND you need to set the "feeasset" key in the config.json.